### ISSUE TITLE
Add LICENSE

### ISSUE THREAD

**Issue by cybardev** [ayunami2000/ayunpictojava#23]
Please consider adding a LICENSE file to the repository to clarify license terms. Maybe GPLv3 or some other FOSS license, if you're open to other people building on the work, or use a custom license to clarify usage rights.

I say this cuz repos without a LICENSE file are implicitly copyrighted according to GitHub ToS. If you want to retain the copyright then that's fine too, but maybe adding a LICENSE file with the copyright info would still be good to clarify intent.

Thanks.

**Comment by cybardev** [comment]
Ok, so according to what I saw on the Discord, here's the situation for reference:
- the UI code is from somewhere else without a license
- the author of the UI code was okay with this derivative
- the author of this app is okay with derivatives
- but it's legally hazy - no one's _probably_ going to sue anyone, but modify/redistribute "at your own risk"

I might try to make my own from scratch later on. For now, I'm keeping my fork (not modifying the UI code) that just containerizes the app and deploys to Render. It's mostly a DevOps exercise for me more than anything anyway.

Closing as I got enough info, and I don't think a license can be added at this point.

**Comment by ayunami2000** [comment]
Yeah that's about right. Sorry I ignored the question I was really busy with the holidays and stuff.

**Comment by cybardev** [comment]
No worries. Life happens. 😄
